### ISSUE TITLE
library like pyvcf to read vcf files

### ISSUE THREAD

**Issue by visze** [PEDIA-Charite/classifier#10]
https://github.com/PEDIA-Charite/classifier/blob/6a930bbca3961efae33f12fdcc32c8aa1f17d77d/lib/get_variant.py#L45

@la60312 maybe it would be easier to use a library like pyvcf to read vcf fies. Would be easier to handle different fields, support multi samples etc.

cheers

**Comment by la60312** [comment]
@visze you are right. I didn't try this library before. Thanks for the suggestion.
